spec/factories/donors.rb

```diff
@@ -16,5 +16,21 @@
         create_list(:donation_with_cause, 2, donor: donor)
       end
     end
+
+    factory :smithie_black do
```

Comment:
Sorry. This is a bit too granular for the factory. We should just pass this as an argument in the test setup. 😀